project/Dependencies.scala

```diff
@@ -24,12 +24,13 @@ object Dependencies {
   val jacksonDatabind: ModuleID =    "com.fasterxml.jackson.core" % "jackson-databind"    % jacksonV
   val jacksonCore: ModuleID =        "com.fasterxml.jackson.core" % "jackson-core"        % jacksonV
 
-  val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.1"
+  val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.2"
   val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
   val scalaLogging: ModuleID =   "com.typesafe.scala-logging" %% "scala-logging"   % scalaLoggingV
   val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "2.2.5"
   val ficus: ModuleID =          "com.iheart"                 %% "ficus"           % "1.4.0"
   val cats: ModuleID =           "org.typelevel"              %% "cats"            % "0.9.0"
+  val httpClient: ModuleID =     "org.apache.httpcomponents"  % "httpclient"       % "4.5.3"  // upgrading a transitive dependency to avoid security warnings
```

Comment:
Should we exclude this from whatever is pulling it in transitively?

Reply:
Looks like from google libraries?
```
[info]   +-org.broadinstitute.dsde.workbench:workbench-google_2.12:0.3-97e266d [S]
[info]   | +-com.google.apis:google-api-services-admin-directory:directory_v1-rev82-1.22.0
[info]   | | +-com.google.api-client:google-api-client:1.22.0
[info]   | |   +-com.google.http-client:google-http-client-jackson2:1.22.0
[info]   | |   | +-com.fasterxml.jackson.core:jackson-core:2.1.3 (evicted by: 2.9.0)
[info]   | |   | +-com.fasterxml.jackson.core:jackson-core:2.9.0
[info]   | |   | +-com.google.http-client:google-http-client:1.22.0
[info]   | |   |   +-com.google.code.findbugs:jsr305:1.3.9 (evicted by: 3.0.0)
[info]   | |   |   +-com.google.code.findbugs:jsr305:3.0.0
[info]   | |   |   +-org.apache.httpcomponents:httpclient:4.0.1
```

Reply:
Do you prefer to have the exclusions in place, or is this an open question?

I generally prefer to NOT add the excludes, unless there's a conflict. I tried `sbt assembly` on this and did not see a conflict. Unless we really need the exclude, I find them to be just more code to maintain.

All that said, I have the exclusions working locally - let me know which you'd prefer!

Reply:
Thanks - given that sbt does the right thing, it's fine to leave out the excludes.